### DISCUSSION TITLE
Hi, I'm running into some problems while running the code, hope to get your answers！

### DISCUSSION THREAD

**Discussion by xsk-code** [colinpcurtis/datares_GANs#12] [Q&A]
When I run to make dataset, the program has been stuck at this step without corresponding。
![SI8C`H54OTTTM~W8{8PU}8C](https://user-images.githubusercontent.com/58896893/159884595-c62d262c-63be-4ce3-acad-31a0cb2581f7.png)

**Comment by colinpcurtis** [comment]
It takes quite a few minutes to run.  I'd let the program run because it has to download a lot of images, which could take a while if your internet is slow.
